### ISSUE TITLE
Camera viewport lock - unclear how to get out of the camera view

### ISSUE THREAD

**Issue by jiajasper** [playcanvas/editor#795]
Clicking on the camera viewport will switch users' viewport to what the camera sees, but it is unclear to users how to get out of the camera view.

<img width="514" alt="image" src="https://user-images.githubusercontent.com/30747200/174441539-ff299e26-ef74-4821-b1b8-5816bab561b4.png">

@jiajasper Ticket to self - UX needs work.

**Comment by jiajasper** [comment]
@leonidaspir thats a great point actually, the link is now only open to internal members. I am unsure if it is safe to share the file to public - but i will make sure to attach screenshots to each link so even the public can preview the changes!

**Comment by yaustar** [comment]
Wouldn't 3 clash with other highlighting functionality that we have (eg highlighting a referenced entity from script attributes)? We also have some drag and drop highlighting that may clash (eg dragging a template into the scene from the assets panel, dragging texture to a material, etc)

**Comment by yaustar** [comment]
For reference, this is the current behaviour of the Editor
<img width="1678" alt="image" src="https://user-images.githubusercontent.com/16639049/175351575-a405ef3e-ee6e-46fb-acf9-7c7979679c99.png">

https://user-images.githubusercontent.com/16639049/175351824-52d36a00-ef32-4f70-9629-e9573e83ab77.mp4

The window is highlighted in the video saying 'you can drag here'

**Comment by jiajasper** [comment]
@yaustar We can skip highlighting camera in the hierarchy as long as the camera dropdown in the viewport is highlighted.

**Comment by yaustar** [comment]
I disagree with this as I would like to be able to position a camera even if it's disabled. Otherwise I would have to enable it, select the camera in the list, move it it and then disable it.
